golang/cmd/factoryinsight/v2/services/service-tags_test.go

```diff
@@ -0,0 +1,98 @@
+package services
+
+import (
+	"fmt"
+	"testing"
+	"time"
+)
+
+func TestTimeBucketToDuration(t *testing.T) {
+	testMinuteString := "3m"
```

Comment:
Please convert these to a map of input & expected output, then use a loop to check all of them, to keep the code more concise.

Reply:
The test uses map and list for inputs/expected results now.